common/persistence/cassandra/cassandraHistoryV2Persistence.go

```diff
@@ -107,6 +113,11 @@ func convertCommonErrors(operation string, err error) error {
 // AppendHistoryNodes upsert a batch of events as a single node to a history branch
 // Note that it's not allowed to append above the branch's ancestors' nodes, which means nodeID >= ForkNodeID
 func (h *cassandraHistoryV2Persistence) AppendHistoryNodes(request *p.InternalAppendHistoryNodesRequest) error {
+	if len(request.Events.Data) > h.transactionSizeLimit {
+		return &p.TransactionSizeError{
+			Msg: "AppendHistoryNodes operation failed, transaction size was too large",
```

Comment:
include requested_tx_size: len(request.Events.Data) in the error string. Same comment in other places where the size limit is checked